src/react/features/connector/hooks/index.js

```diff
@@ -0,0 +1,13 @@
+import { useGetHome } from './useGetHome'
+import { useGetRoom } from './useGetRoom'
+import { useSaveHome } from './useSaveHome'
+import { useSaveRoom } from './useSaveRoom'
+
+export { useAddRoom } from './useAddRoom'
+
+export const useChannel = () => {
+  useGetHome()
+  useSaveHome()
```

Comment:
I wonder why we need to "save home" at startup? 

"save room" we need because room is "created" by reading the folder if diograph.json is not available but isn't home always just home.json. So at startup we just read the home.json and save the same home.json again?

Reply:
It says useSaveHome, not save home :) Basically, it initializes the subscription to the home and room changes that triggers the the actual save action.